docs/user/datastores/data.rst

```diff
@@ -0,0 +1,33 @@
+.. _data_migration:
+
+Moving and Migrating Data
+=========================
+
+If you want an offline copy of your data, or you want to move data between networks, you can
+export compressed Avro files containing your simple features. To do this using the command line
+tools, use the export command with the ``format`` and ``gzip`` options:
+
+.. code-block:: bash
+
+    $ geomesa-accumulo export ... -f mySft --format avro --gzip 6 -o myFeatures.avro
```

Comment:
Do we have a good way to indicate that 'geomesa-accumulo' is the name of the Accumulo command and that folks can expect to have a similar command for their other datastores?

Reply:
I'll add a comment